hardhat.config.js

```diff
@@ -41,29 +42,17 @@ module.exports = {
     },
   },
   networks: {
-    hardhat: {
-      chainId: 31337,
-    },
-    ganache: {
+    local: {
       url: `http://${PROVIDER_HOST}:${PROVIDER_PORT}`,
-      network_id: 31337,
+      chainId: 31337,
       logger: console,
     },
-    // More about networks config:
-    // https://hardhat.org/config/#json-rpc-based-networks
-    goerli: {
+    mumbai: {
       url: PROVIDER_URL || '',
       accounts: { mnemonic: MNEMONIC },
       chainId: ENV_CHAIN_IDS[NETWORK],
       gas: 7700000,
-      gasPrice: 1 * GWEI,
-    },
-    matic_mumbai_testnet: {
-      url: PROVIDER_URL || '',
-      accounts: { mnemonic: MNEMONIC },
-      chainId: ENV_CHAIN_IDS[NETWORK],
-      gas: 7700000,
-      gasPrice: 1 * GWEI,
+      gasPrice: 30 * GWEI,
```

Comment:
do we need to hardcode gasprice?

Reply:
> gasPrice: Its value should be "auto" or a number. This parameter behaves like gas. Default value: "auto".

Lets set it to 'auto'.

Reply:
yes

Reply:
I've removed `gas` and `gasPrice`. Hardhat will set these by default to 'auto'.